.github/workflows/therock_test_harness.yml

```diff
@@ -0,0 +1,98 @@
+name: TheRock Test Harness
```

Comment:
None of our other workflows use "TheRock" in their names or file names. Could choose a different name for this.